tools/osv-scanner/cmd/osv-scanner/config.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+)
+
+type Config struct {
+	IgnoredVulnIds []string
+}
+
+// TryLoadConfig tries to load config in `target` or any of it's parent dirs
+// `target` will be the key for the entry in configMap
+// Will shortcut and return "" if globalConfig is not nil
+func TryLoadConfig(target string, configMap map[string]Config) string {
+	if globalConfig != nil {
+		return ""
+	}
+	stat, err := os.Stat(target)
+	if err != nil {
+		log.Fatalf("Failed to stat path: %s", err)
+	}
+
+	if stat.IsDir() && !strings.HasSuffix(target, "/") {
+		// Make sure directories ends with '/'
+		target += "/"
```

Comment:
I can see benefit in aiming to make the scanner platform agnostic. To this end, consider using https://pkg.go.dev/path/filepath#pkg-constants rather than hardcoding Unix-specific filesystem constructs.